website/docs/docs/collaborate/git-version-control.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "About git"
+id: git-version-control
+description: "Git overview"
+sidebar_label: "About git"
+---
+
+A [version control](https://en.wikipedia.org/wiki/Version_control) system allows you and your teammates to work collaboratively, safely, and simultaneously on a single project. Version control helps you track all the code changes made in your dbt project.
+
+In a distributed version control system, every developer has a full copy of the project and project history. Git is one of the most popular distributed version control systems and is commonly used for both open source and commercial software development, with great benefits for individuals, teams and businesses.
+
+Git allows developers see the entire timeline of their changes, decisions, and progression of any project in one place. From the moment they access the history of a project, the developer has all the context they need to understand it and start contributing.
+
+When you develop in the Cloud IDE, you can leverage Git directly to version control your code from your browser. This means you can branch, commit, push, and pull code with a couple of clicks.
```

Comment:
Could we have a blurb for dbt Core / command line as well? it also supports Git

Reply:
good callout, what about the new change?